pkgs/development/tools/jdt-ls/default.nix

```
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchurl
+, jdk
+, bash
+}:
+
+stdenv.mkDerivation rec {
+  pname = "jdt-ls";
+  version = "0.67.0";
+  timestamp = "202012170459";
+
+  src = fetchurl {
+    url = "https://download.eclipse.org/jdtls/milestones/${version}/jdt-language-server-${version}-${timestamp}.tar.gz";
+    sha256 = "11qzwf2l737hdv1g7bfcrmdxcrl7g9z2k9nhy9lknlabmhq4mcdl";
+  };
+
+  # After running unpackPhase, the generic builder changes the current directory
+  # to the directory created by unpacking the sources. If there are multiple
+  # source directories, you should set sourceRoot to the name of the intended
+  # directory.
+  sourceRoot = ".";
+
+  buildInputs = [
+    bash
+    jdk
+  ];
+
+  installPhase =
+    let
+      # There are config_ss_* directories too, use them to run the lightweight
+      # version, see
+      # https://github.com/eclipse/eclipse.jdt.ls/wiki/Running-the-lightweight-syntax-language-server
+      config-path = if stdenv.isDarwin then "config_mac" else "config_linux";
```

I think it looks very good overall and should be merged, but should use camel case here:
```suggestion
      configPath = if stdenv.isDarwin then "config_mac" else "config_linux";
```